tests/syntax/test_regions.py

```diff
@@ -33,16 +33,15 @@ def test_circular_in():
 
 def test_circular_lazy():
     ego = sampleEgoFrom("""
-        vf = VectorField("Foo", lambda pos: 2 * pos.x)
-        x = 0 relative to vf
-        ego = Object at Range(-1, 1) @ 0, with foo CircularRegion(0@0, x)
+    x = relative heading of 61 deg from Range(30,59) deg
```

Comment:
This isn't a lazy value (only a random value), defeating the purpose of the test. You should be able to use the old code with `x.yaw` (maybe restricting the range of `pos.x` to avoid angle normalization issues).